src/Core/functions.php

```diff
@@ -51,7 +51,7 @@ function h($text, $double = true, $charset = null)
             } else {
                 $text = '(object)' . get_class($text);
             }
-        } elseif (is_bool($text) || is_null($text) || is_int($text)) {
+        } elseif (is_bool($text) || is_int($text) || $text === null) {
```

Comment:
If you want to do the nano opt here, then the null check should be first, not last :)

Reply:
Ideally in separate PR addressing similar cases around the codebase.

Reply:
OK, gimme a sec